### PR TITLE
Update starter kit

### PR DESCRIPTION
**Pull request by serudda**
## Changes Made 🎉

- [x] feat: added new feature to improve user experience
- [ ] fix: corrected a bug with login functionality
- [x] refactor: improved code readability and organization
- [ ] docs: updated README with new instructions
- [x] chore: updated dependencies and configuration files

### Describe Changes
- We replaced the Discord provider with the Twitter provider to provide an example that is easier to visualize.
- All the necessary logic was added to be able to include Lemon Squeezy within an initial project.
- All logic related to i18-next was removed in order to simplify the Starter Kit.
- The project was cleaned up in general.

## Checklist ✅

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: .env.local.example [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+# Prisma
+# https://www.prisma.io/docs/reference/database-reference/connection-urls#env
+# The [PASSWORD] is generated into Supabase platform (https://app.supabase.com/project/)
+# WITH SUPABASE
+# DATABASE_URL="postgresql://postgres:[PASSWORD]@db.elnalpjubvyknvqaawva.supabase.co:5432/postgres"
+# WITH DOCKER - DB
+# DATABASE_URL="postgresql://root:admin@localhost:5432/started-db?schema=public"
+# WITH DOCKER - APP/DB
+# DATABASE_URL="postgresql://root:admin@localhost:5432/started-db?schema=public"
```
Comment (Zyruks) [inline .env.local.example:25]:
```suggestion

```

Duplicado con linea 13

File: apps/nextjs/src/styles/globals.css [hunk 2]
```diff
@@ -1,3 +1,127 @@
...
+    circle,
+    var(--tw-gradient-stops) var(--dots-size),
+    var(--tw-gradient-stops) var(--dots-size),
+    transparent var(--dots-size),
+    transparent 100%
+  );
+
+  background-size: var(--dots-separation);
+  mask-image: var(--mask-image);
+}
```
Comment (Zyruks) [inline apps/nextjs/src/styles/globals.css]:
```suggestion

```


La clase dotted-background está de más, ya que funciona con un componente que no está en el template.

File: apps/nextjs/src/styles/globals.css [hunk 3]
```diff
@@ -1,3 +1,127 @@
...
+  .rotate-y-0 {
+    transform: rotateY(0);
+  }
+  .rotate-y-90 {
+    transform: rotateY(90deg);
+  }
+  .-rotate-y-90 {
+    transform: rotateY(-90deg);
+  }
+}
```
Comment (Zyruks) [inline apps/nextjs/src/styles/globals.css]:
```suggestion
/************************************************/
/*                 UTILITY CLASSES              */
/************************************************/

@layer utilities {

}
```

Igual acá esta demás esta utility, ya que tenemos `tailwindcss-animated`

**Review by Zyruks — APPROVED** [review]
Está listo, solo queda eliminar unas cuantas que están de más.

File: .env.local.example (hunk 1 above)
Comment (Zyruks) [reply .env.local.example:25]: Done

File: apps/nextjs/src/styles/globals.css (hunk 2 above)
Comment (serudda) [reply apps/nextjs/src/styles/globals.css]: DONE

File: apps/nextjs/src/styles/globals.css (hunk 3 above)
Comment (serudda) [reply apps/nextjs/src/styles/globals.css]: DONE

**Pull request merged**